pkgs/by-name/ca/caddy/plugins.nix

```diff
@@ -69,8 +98,18 @@ caddy.overrideAttrs (
           fi
         done
       done < <($out/bin/caddy build-info)
+
       if (( ''${#notfound[@]} )); then
-        >&2 echo "Plugins not found: ''${notfound[@]}"
+        for plugin in "''${notfound[@]}"; do
+          base=''${plugin%@*}
+          specified=''${plugin#*@}
+          while read kind module expected; do
+            [[ "$kind" = "dep" && "$module" = "$base" ]] || continue
+            echo "Plugin \"$base\" have incorrect tag:"
+            echo "  specified: \"$base@$specified\""
+            echo "  got: \"$base@$expected\""
+          done < <($out/bin/caddy build-info)
+        done
```

Comment:
Does not display anything if a plugin is not found at all.

Reply:
Added a fall back but I don't think this will ever get triggered unless there's smth wrong with xcaddy